modules/manage/pages/iceberg/iceberg-topics-databricks-unity.adoc

```diff
@@ -0,0 +1,160 @@
++
+.Create a new topic and set `redpanda.iceberg.mode`:
+[,bash]
+----
+rpk topic create <topic-name> --topic-config=redpanda.iceberg.mode=key_value
+----
++
+.Set `redpanda.iceberg.mode` for an existing topic:
+[,bash]
+----
+rpk topic alter-config <topic-name> --set redpanda.iceberg.mode=key_value
+----
+
+. Produce to the topic. For example, 
++
+[,bash]
+----
+echo "hello world\nfoo bar\nbaz qux" | rpk topic produce <topic-name> --format='%k %v\n'
+----
+
+You should see the topic as a table in Unity Catalog.
+
+. In Catalog Explorer, open your catalog. You should see a `redpanda` schema, in addition to `default` and `information_schema`.
+. The `redpanda` schema and the table residing within this schema are automatically added for you. The table name is the same as the topic name. 
+
+== Query Iceberg table using Databricks SQL
+
+You can query the Iceberg table using different engines, such as Databricks SQL, PyIceberg, or Apache Spark. To query the table or view the table data in Catalog Explorer, ensure that your account has the necessary permissions to read the table. Review the official Databricks documentation on https://docs.databricks.com/aws/en/data-governance/unity-catalog/manage-privileges/?language=SQL#grant-permissions-on-objects-in-a-unity-catalog-metastore[granting permissions to objects^] and https://docs.databricks.com/aws/en/data-governance/unity-catalog/manage-privileges/privileges[Unity Catalog privileges^] for details.
+
+The following example shows how to query the Iceberg table using SQL in Databricks SQL.
+
+. In the Databricks console, open *SQL Editor*.
+. In the query editor, run the following:
++
+[,sql]
+----
+-- Ensure that the catalog and table name are correctly parsed in case they contain special characters
+SELECT * FROM `<catalog-name>`.redpanda.`<table-name>`;
+----
++
+Your query results should look like the following:
++
+[,sql,role="no-copy no-wrap"]
+----
+# Example for redpanda.iceberg.mode=key_value with 1 record produced to topic
++--------------------------------------------------------------------------+------------+-------------------------+
+| redpanda                                                                 | value      | redpanda.timestamp_hour |
++--------------------------------------------------------------------------+------------+-------------------------+
+| {"partition":0,"offset":"0","timestamp":"2025-04-02T18:57:11.127Z",      | 776f726c64 | 2025-04-02-18           |
+| "headers":null,"key":"68656c6c6f"}                                       |            |                         |
++--------------------------------------------------------------------------+------------+-------------------------+
+----
+
```

Comment:
Is there anything we should mention in this doc regarding metadata refresh? Does Unity automatically refresh table metadata?